src/main/java/edu/wpi/grip/ui/util/SearchUtility.java

```diff
@@ -0,0 +1,39 @@
+package edu.wpi.grip.ui.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Utility for fuzzy text searching
+ */
+public class SearchUtility {
+
+    /**
+     * @return true if query if approximately a substring of text, not taking into account capitalization, whitespace,
+     * or minor typos
+     */
+    public static boolean fuzzyContains(String text, String query) {
+        if (query.isEmpty()) return true;
+        if (text.isEmpty()) return false;
+
+        // Normalize the capitalization and whitespace (which are currently both pretty inconsistent among operations)
+        text = text.toUpperCase().replaceAll("[^a-zA-Z]", "");
+        query = query.toUpperCase().replaceAll("[^a-zA-Z]", "");
```

Comment:
See the comment right before this line. The operations are currently very inconsistent about whitespace and other non-alpha characters. If you search "bitwiseAnd", "bitwise and", or "bitwise_and", it should return the same results.

I'm not sure that a quote syntax is a good idea.  How would people know to use that syntax?  And what situation would the regular fuzzy search not work?